packages/@blockprotocol/type-system-node/tests/uri.test.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseUri, VersionedUri } from "@blockprotocol/type-system-node";
+
+describe("BaseURI Class", () => {
```

Comment:
`'describe' is not defined` can be fixable via top-level `.eslntrc`. Try finding `overrides` and adding this item there:

```
    {
      files: ["packages/**/*.test.{j,t}s{x,}"],
      env: {
        jest: true,
        node: true,
      },
    },
```

Reply:
Thanks! That was stumping me